doc/development-guidelines/documentation-guide.rst

```diff
@@ -28,7 +28,45 @@ The following steps will explain how to use it.
 Once those three questions are answered then a brand new doc folder will be added to the CSC.
 Inside of the files, there are comments that explain how to fill out each section in detail.
 
+The conf.py file is a Sphinx configuration file that contains information on Sphinx should build the site as well as any extensions that should be loaded. 
+You can find the template conf.py at https://github.com/lsst-ts/cookiecutter_tssw_csc_doc/blob/90c866162936f4d31d5fb1e1684e17a75128765d/%7B%7Bcookiecutter.doc%7D%7D/conf.py.
+Documenteer is a tool that provides pre-made Sphinx configurations for 3 different purposes that SQRE provides the project.
+
+1. pipelines
+2. guide
+3. technote
+
+It also provides several project extensions for the RSP packages.
+All documentation is available at https://documenteer.lsst.io.
+
+..
+    Remove with DM-44045.
+
+The template does not yet provide compatibility with documenteer 1.0 and above.
+See the :ref:`development-guidelines/documentation-guide:Switching to the guide theme` section for how to accomplish this.
+
 Publishing lsst.io Site
 =======================
 
 This functionality is included in the DevelopPipeline for the Jenkins Shared Library.
+
+Switching to the guide theme
```

Comment:
Please rephrase this section header to not refer to changes. Probably `Documenteer` would be sufficient but if you think of a better header then I'm open for it.